examples/gw-routers/k8s/gateway-right.yaml

```diff
@@ -50,6 +50,6 @@ data:
       labels:
         app: "gw-right"
       ipam:
```

Comment:
remove changes to this file... it's unrelated to our NSE... the stuff under vl3_basic